libs/components/forms/src/assets/locales/resources_en_US.json

```diff
@@ -7,6 +7,42 @@
     "_description": "Screen reader only label for character count over limit symbol",
     "message": "You are over the character limit."
   },
+  "skyux_form_error_character_count": {
+    "_description": "Error message for a field with a value that exceeds the character count limit",
+    "message": "Limit {0} to {1} character(s)."
+  },
+  "skyux_form_error_date": {
+    "_description": "Error message for a field with an invalid date value",
+    "message": "Select or enter a valid date."
+  },
+  "skyux_form_error_email": {
+    "_description": "Error message for a field with an invalid email value",
+    "message": "Enter an email address with a valid format."
+  },
+  "skyux_form_error_maxlength": {
+    "_description": "Error message for a field with a value that is too long",
+    "message": "Limit {0} to {1} character(s)."
+  },
+  "skyux_form_error_minlength": {
+    "_description": "Error message for a field with a value that is not long enough",
+    "message": "{0} must be at least {1} character(s)."
```

Comment:
are screenreaders able to parse `character(s)` in a non awkward way?

Reply:
Our resources don't currently have a better way to handle pluralization. This was moved over from [skyux_input_box_error_maxlength](https://github.com/blackbaud/skyux/blob/main/libs/components/forms/src/assets/locales/resources_en_US.json#L128).

Reply:
goootcha, ima throw a flex topic on the board just for myself but resolve this thread